examples/plot_simulate_somato.py

```diff
@@ -4,8 +4,15 @@
 ================================================
 
 This example demonstrates how to calculate the inverse solution of the median
-nerve evoked response in S1 from the MNE somatosensory dataset, and then
-simulate a biophysical model network that reproduces the observed dynamics.
+nerve evoked response potential (ERP) in S1 from the MNE somatosensory dataset,
+and then simulate a biophysical model network that reproduces the observed
+dynamics. Note that we do not expound on how we came up with the sequence of
+evoked drives added to the HNN model to simulate the median nerve evoked
+response, rather, this example demonstrates its implementation. For those who
+want more background on the HNN model and the process used to articulate the
+proximal and distal drives needed to simulate evoked responses, see the
+`HNN ERP tutorial`_ and prior studies that used HNN to elucidate the circuit
+mechanisms responsible for the tactile evoked response [1]_, [2]_.
```

Comment:
@stephanie-r-jones what do you think of this? I've also added a segment at L159-L162 to again direct users to the tutorial/references should they want to know more about the general process of assigning evoked drives.

Reply:
A few suggestions: 
1) Rather than saying this simulates the median nerve response, specify that this simulates the current dipole response source localized to the hands area of SI to median nerve stimulation using the HNN template neocortical column model. 
2) State that this modeling example and the parameter use to generate it are not part of a current publication, but motivated by prior publications [1], [2]... 
3) Does this example plot the source localized data and model simulation overlaid? Or do any type of direct comparison between them? Is the dipole scaled to match the data?  If so, there is more to explain here.

Reply:
> A few suggestions:
> 
>     1. Rather than saying this simulates the median nerve response, specify that this simulates the current dipole response source localized to the hands area of SI to median nerve stimulation using the HNN template neocortical column model.

Something similar to what you're suggesting is actually stated in L6-L8. Note that the method we used to extract a "characteristic" dipole response from the post-central gyrus computes the 1st principle component over all vertices and is therefore not necessarily localized to the hand area of SI. (See #253 around [this comment](https://github.com/jonescompneurolab/hnn-core/pull/253#issuecomment-769327281) for the discussion that lead us to use MNE-python's PCA-flipped time course method.)

>     2. State that this modeling example and the parameter use to generate it are not part of a current publication, but motivated by prior publications [1], [2]...

I'll update this!

>     3. Does this example plot the source localized data and model simulation overlaid? Or do any type of direct comparison between them? Is the dipole scaled to match the data?  If so, there is more to explain here.

The empirical and modeled waveforms are [plotted together](https://jonescompneurolab.github.io/hnn-core/stable/auto_examples/plot_simulate_somato.html#sphx-glr-auto-examples-plot-simulate-somato-py); however, we don't present or discuss any quantitative or qualitative comparisons between them.

Reply:
> All sounds good. One further questions, was a scaling factor applied to the model results match the dipole amplitude? If so, can you point me to where this is in the code and there should be an explanation of it somewhere?

Yes, a scaling factor is used. I just pushed https://github.com/jonescompneurolab/hnn-core/pull/283/commits/c195634f710b2612fe2ded5dd451a9f94bc4ccd8 where we explicitly define the smoothing window size and scaling factor in the example (as opposed to merely reading them in from a param file) and briefly explain their purpose. Is this what you were thinking?